pkg/containerspec/containerspec.go

```diff
@@ -38,7 +38,7 @@ type PaastaContainerSpec struct {
 	CPU       *KubeResourceQuantity `json:"cpus"`
 	Memory    *KubeResourceQuantity `json:"mem"`
 	Disk      *KubeResourceQuantity `json:"disk"`
-	DiskLimit *KubeResourceQuantity `json:"disk_limit"`
+	DiskLimit *KubeResourceQuantity `json:"disk_limit,omitempty"`
```

Comment:
I don't know why it's necessary, but last time I checked, the documentation for the operator-sdk said to add `// optional` above the line with omitempty. I think it is needed by the deepcopy generator tool.

You can check the definition of types in the flink-operator

Reply:
Nevermind, here the deepcopy is manually defined, not autogenerated.